doc/install.md

```diff
@@ -233,7 +237,11 @@ from Python.
 To ensure that PyGMT and its dependencies are installed correctly, run the following
 in your Python interpreter:
 
-```python
+```{code-cell} ipython
+---
+tags: [remove-stdout]
+---
+
 import pygmt
 pygmt.show_versions()
 
```

Comment:
> These lines are cell-level metadata for controlling the format of the output. Usually it's not necessary. Here remove-stdout is necessary to hide the long output from pygmt.show_versions().

Would it be nicer to actually show the output of `pygmt.show_versions()` actually? Suggestion to:

1. Split into two code blocks, one for `pygmt.show_versions()` and one for `fig = pygmt.Figure()` below
2. Collapse the output of `pygmt.show_versions()` in a dropdown using the `hide-output` tag (see https://myst-nb.readthedocs.io/en/v0.13.2/use/hiding.html#hide-code-cells).

Reply:
Done in 609d759e8d45e031ebef61376d0537379f0b3f1b and looks better.

Reply:
Readthedocs is showing the PyGMT version as `v0.1.dev64+g7a38068.d20240812`, but I guess that's ok :laughing: